doc/changes/changes_1.0.0.md

```diff
@@ -1,24 +1,41 @@
-# Kafka Connector Extension 1.0.0, released 2020-12-DD
+# Kafka Connector Extension 1.0.0, released 2021-05-06
 
-## Features / Improvements
+Code name: v1.0.0 - Support for JSON Records import and offset reset strategy
+
+## Summary
+
+The release `1.0.0` added support for importing JSON records and consumer offset
+reset strategy. By using offset strategy consumer imports new records after
+topic retention.
```

Comment:
This is not a good idea to divide the summary into multiple lines because it will go like this to the community portal. I think it could look strange there.

Reply:
Fixed.